src/utils.ts

```diff
@@ -114,8 +114,10 @@ export async function buildCeramic (configObj, ipfs?: IpfsApi): Promise<CeramicA
         }
         const [modules, params] = await Ceramic._processConfig(ipfs, ceramicConfig)
         if (configObj.s3StateStoreBucketName) {
-            const s3StateStore = new S3StateStore(configObj.s3StateStoreBucketName)
+            const bucketName = `${configObj.s3StateStoreBucketName}`
```

Comment:
why does this have to be saved in bucket name and why does it have to be in backticks?

Reply:
this was just so we could log it, removed!